test/sumoLogger.test.js

```diff
@@ -1,589 +1,595 @@
+jest.mock('superagent', () => {
```

Comment:
Can this file be indented with 4 spaces please?  I can see the linting is failing in Travis with these errors and a few others

Reply:
Yes, thank you. I will run the autofix.